.npmignore

```diff
@@ -1,3 +1,4 @@
 .nyc_output
 coverage
-benchmark
\ No newline at end of file
+benchmark
+node_modules
```

Comment:
The test file, the rollup config and the editorconfig etc. should also be ignored.